portfolio/src/main/webapp/walkthrough/script.js

```diff
@@ -1,11 +1,20 @@
 async function fetchWalkthroughData() {
-    const fetchedDataResponse = await fetch("/data");
+    const commentLimit = document.getElementById("comment_limit").value;
+    const fetchedDataResponse = await fetch(`/data?comment_limit=${commentLimit}`);
     const fetchedDataJSON = await fetchedDataResponse.json();
     const contentList = document.getElementById("walkthrough-content");
+    contentList.innerHTML = "";
     for (let i = 0; i < fetchedDataJSON.length; i++) {
         let listItem = document.createElement("li");
         listItem.innerText = fetchedDataJSON[i];
         contentList.appendChild(listItem);
     }
 }
 
+async function deleteComments() {
+  const response = await fetch('/delete-data', {
+    method: "POST"
```

Comment:
and if you use my suggestion above, this would be a delete method which is more suitable for a "deleteComments"

:)